### DISCUSSION TITLE
Please tell me the product number for Flat head screw M1.7

### DISCUSSION THREAD

**Discussion by jastat** [Yowkees/keyball#216] [Q&A]
I have stripped the screw thread while assembling keyball61's trackball. I would like to purchase only the screw. Please tell me the product number for Flat head screw M1.7.

**Comment by Yowkees** [comment]
Hi. Thank you for choosing Keyball61!

Product name of Flathead M1.7 screw is FXB-1750E.

I hope you can order from here,
https://wilco.jp/products/F/FXB-E.html
